test/platform/events/streaming.spec.ts

```diff
@@ -9,93 +9,51 @@ import {
-    const event: StreamingKeyRevokeEvent = {
-      type: Events.Type.STREAMING,
-      subType: Events.SubType.Streaming.STREAMING_KEY_REVOKE,
-      key: 'key',
-      timestamp: 1,
-      metadata: {
-        position: '0,0',
-        worldName: 'world-name',
-        host: 'host-name'
-      }
-    }
-
-    expect(StreamingKeyRevokeEvent.validate(event)).toEqual(true)
-    expect(StreamingKeyRevokeEvent.validate(null)).toEqual(false)
-    expect(StreamingKeyRevokeEvent.validate({})).toEqual(false)
-  })
-
-  it('StreamingKeyExpiredEvent static tests must pass', () => {
-    const event: StreamingKeyExpiredEvent = {
-      type: Events.Type.STREAMING,
-      subType: Events.SubType.Streaming.STREAMING_KEY_EXPIRED,
-      key: 'key',
-      timestamp: 1,
-      metadata: {
-        position: '0,0',
-        worldName: 'world-name',
-        host: 'host-name'
-      }
-    }
-
-    expect(StreamingKeyExpiredEvent.validate(event)).toEqual(true)
-    expect(StreamingKeyExpiredEvent.validate(null)).toEqual(false)
-    expect(StreamingKeyExpiredEvent.validate({})).toEqual(false)
-  })
-
-  it('StreamingTimeExceededEvent static tests must pass', () => {
-    const event: StreamingTimeExceededEvent = {
-      type: Events.Type.STREAMING,
-      subType: Events.SubType.Streaming.STREAMING_TIME_EXCEEDED,
-      key: 'key',
-      timestamp: 1,
-      metadata: {
-        position: '0,0',
-        worldName: 'world-name',
-        host: 'host-name'
-      }
+  const testCases = [
+    {
+      name: 'StreamingKeyResetEvent',
+      eventClass: StreamingKeyResetEvent,
+      subType: Events.SubType.Streaming.STREAMING_KEY_RESET
+    },
+    {
+      name: 'StreamingKeyRevokeEvent',
+      eventClass: StreamingKeyRevokeEvent,
+      subType: Events.SubType.Streaming.STREAMING_KEY_REVOKE
+    },
+    {
+      name: 'StreamingKeyExpiredEvent',
+      eventClass: StreamingKeyExpiredEvent,
+      subType: Events.SubType.Streaming.STREAMING_KEY_EXPIRED
+    },
+    {
+      name: 'StreamingTimeExceededEvent',
+      eventClass: StreamingTimeExceededEvent,
+      subType: Events.SubType.Streaming.STREAMING_TIME_EXCEEDED
+    },
+    {
+      name: 'StreamingPlaceUpdatedEvent',
+      eventClass: StreamingPlaceUpdatedEvent,
+      subType: Events.SubType.Streaming.STREAMING_PLACE_UPDATED
     }
-
-    expect(StreamingTimeExceededEvent.validate(event)).toEqual(true)
-    expect(StreamingTimeExceededEvent.validate(null)).toEqual(false)
-    expect(StreamingTimeExceededEvent.validate({})).toEqual(false)
-  })
-
-  it('StreamingPlaceUpdatedEvent static tests must pass', () => {
-    const event: StreamingPlaceUpdatedEvent = {
-      type: Events.Type.STREAMING,
-      subType: Events.SubType.Streaming.STREAMING_PLACE_UPDATED,
-      key: 'key',
-      timestamp: 1,
-      metadata: {
-        position: '0,0',
-        worldName: 'world-name',
-        host: 'host-name'
+  ]
+
+  testCases.forEach(({ name, eventClass, subType }) => {
```

Comment:
It might be better to use the jest API to make these tests run by using `test.each` but this works as well.